test/unit/math/fwd/scal/fun/nan_util.hpp

```diff
@@ -15,7 +15,7 @@ void test_nan_fd(const F& f, const double& arg1, const bool& throws) {
     EXPECT_THROW(f(arg1_v), std::domain_error);
   } else {
     EXPECT_TRUE(stan::math::is_nan(f(arg1_v).val_));
-    EXPECT_TRUE(stan::math::is_nan(f(arg1_v).d_));
+    // EXPECT_TRUE(stan::math::is_nan(f(arg1_v).d_));
```

Comment:
These test should just be removed and not commented out if they are no longer needed.

Reply:
OK.  Done.  (This whole file will go away when the tests are all ported over to the framework, which has NaN checks built-in.)